src/spmBatching/setBatchRealign.m

```diff
@@ -32,31 +46,68 @@
+            if strcmp(action, 'realignUnwarp')
+                matlabbatch{end}.spm.spatial.realignunwarp.data(1, runCounter).pmscan = '';
+                matlabbatch{end}.spm.spatial.realignunwarp.data(1, runCounter).scans = file;
+            else
+                matlabbatch{end}.spm.spatial.realign.estwrite.data{1, runCounter} = file;
+            end
 
             runCounter = runCounter + 1;
         end
 
     end
 
+    %% defaults
     % The following lines are commented out because those parameters
     % can be set in the spm_my_defaults.m
+
+    % -----------------
+    % REALIGN
+    % REALIGN & RESLICE
+    % -----------------
+    %
     % matlabbatch{end}.spm.spatial.realign.estwrite.eoptions.quality = 1;
     % matlabbatch{end}.spm.spatial.realign.estwrite.eoptions.sep = 2;
     % matlabbatch{end}.spm.spatial.realign.estwrite.eoptions.fwhm = 5;
-    % matlabbatch{end}.spm.spatial.realign.estwrite.eoptions.rtm = 1;
+    % matlabbatch{end}.spm.spatial.realign.estwrite.eoptions.rtm = 0;
     % matlabbatch{end}.spm.spatial.realign.estwrite.eoptions.interp = 2;
     % matlabbatch{end}.spm.spatial.realign.estwrite.eoptions.wrap = [0 0 0];
-    % matlabbatch{end}.spm.spatial.realign.estwrite.roptions.which = [0 1];
+    %
     % matlabbatch{end}.spm.spatial.realign.estwrite.roptions.interp = 3;
     % matlabbatch{end}.spm.spatial.realign.estwrite.roptions.wrap = [0 0 0];
     % matlabbatch{end}.spm.spatial.realign.estwrite.roptions.mask = 1;
 
+    % ----------------
+    % REALIGN & UNWARP
+    % ----------------
+    %   matlabbatch{1}.spm.spatial.realignunwarp.eoptions.quality = 1;
+    %   matlabbatch{1}.spm.spatial.realignunwarp.eoptions.sep = 2;
+    %   matlabbatch{1}.spm.spatial.realignunwarp.eoptions.fwhm = 5;
+    %   matlabbatch{1}.spm.spatial.realignunwarp.eoptions.rtm = 0;
+    %   matlabbatch{1}.spm.spatial.realignunwarp.eoptions.einterp = 2;
+    %   matlabbatch{1}.spm.spatial.realignunwarp.eoptions.ewrap = [0 0 0];
+    %
+    %   matlabbatch{1}.spm.spatial.realignunwarp.uweoptions.basfcn = [12 12];
+    %   matlabbatch{1}.spm.spatial.realignunwarp.uweoptions.regorder = 1;
+    %   matlabbatch{1}.spm.spatial.realignunwarp.uweoptions.lambda = 100000;
+    %   matlabbatch{1}.spm.spatial.realignunwarp.uweoptions.jm = 0;
+    %   matlabbatch{1}.spm.spatial.realignunwarp.uweoptions.fot = [4 5];
+    %   matlabbatch{1}.spm.spatial.realignunwarp.uweoptions.sot = [];
+    %   matlabbatch{1}.spm.spatial.realignunwarp.uweoptions.uwfwhm = 4;
+    %   matlabbatch{1}.spm.spatial.realignunwarp.uweoptions.rem = 1;
+    %   matlabbatch{1}.spm.spatial.realignunwarp.uweoptions.noi = 5;
+    %   matlabbatch{1}.spm.spatial.realignunwarp.uweoptions.expround = 'Average';
+    %
+    %   matlabbatch{1}.spm.spatial.realignunwarp.uwroptions.rinterp = 3;
+    %   matlabbatch{1}.spm.spatial.realignunwarp.uwroptions.wrap = [0 0 0];
+    %   matlabbatch{1}.spm.spatial.realignunwarp.uwroptions.mask = 1;
+
```

Comment:
realign & unwarp is.... now set for native space?

Reply:
I am not sure what you mean. :thinking:

Reply:
my bad.. my mind went to something else!